### ISSUE TITLE
Syntax Highlighting on Code Blocks not working

### ISSUE THREAD

**Issue by ashokgelal** [42Squares/LightPaper#19]
_From @susanBuck on July 17, 2014 19:57_

Is there a trick to get the syntax highlighting on code blocks to work? 

Not seeing it - [screensot](http://note.io/1mlxznL)

Using Version 0.9 

Syntax highlighting is enabled in prefs: [screenshot](http://note.io/1mlxzn)

_Copied from original issue: AwsmApps/LightPaper-Support#33_

**Comment by ashokgelal** [comment]
_From @deadlyicon on July 18, 2014 16:51_

Doesn't work for me either

**Comment by ashokgelal** [comment]
This is a known issue. The solution for now is to append a `lang-` in front of the language name:
```
```lang-php
$foo = 'bar'
function foobar(){
	return 'baz';
}
```

**Comment by ashokgelal** [comment]
_From @susanBuck on July 19, 2014 6:56_

Ah gotcha. Unfortunately that syntax breaks highlighting when viewed on github.com.

Thanks for the info! 

side note - I dug around for this for a while, is it documented somewhere?

**Comment by ashokgelal** [comment]
Unfortunately no. I've not done a good job of documenting some of the nice features of LightPaper. Hopefully someone will standup and send me a pull request :)

**Comment by ashokgelal** [comment]
_From @CrashyBang on December 18, 2014 4:1_

:+1:

**Comment by ashokgelal** [comment]
_From @pavelloz on June 29, 2015 8:59_

Is there any chance you could unify the prismjs API within LightPaper with github markdown api?

It would be a killer feature for me to have a document that I can write in LightPaper and just copy over to the wiki at github.

**Comment by ashokgelal** [comment]
FYI, this has been resolved in the latest ver 1.0. No need to add `lang-` anymore. 

Here is a list of new features, improvements, and bug fixes in version 1.0: [Release Notes v1.0](https://github.com/42Squares/LightPaper/blob/master/ReleaseNotes%20-%201.0.md).
